lib/wallet.py

```diff
@@ -125,11 +125,12 @@ def find_utxos_for_privkey(txin_type, privkey, compressed):
         raise BaseException(_('No inputs found. (Note that inputs need to be confirmed)'))
     total = sum(i.get('value') for i in inputs)
     if fee is None:
+        if not network.config.has_fee_estimates():
+            raise BaseException('Dynamic fee estimates not available')
```

Comment:
Not sure why this change. You forgot to handle the case where the user has dynamic fees disabled and there are no fee estimates available. He should be able to sweep none-the-less (as he is using static fees anyway). The line with `config.fee_per_kb()` below, that you have deleted, handled that.

Reply:
I was actually not able to use `config.fee_per_kb()` the way it was used before, probably because the `sweep` function was taken out of the `Abstract_Wallet` class in a recent commit. So I just used `network.config.has_fee_estimates()` while over looking the case that you correctly stated.